README.md

```diff
@@ -184,8 +260,20 @@ backend.start();
 const app = createApp({
 ```
 
+`copy paste`
+
+```.tsx
+import {
+  defaultColumnsWithAutogovStatusRightOf,
+  AutogovLatestReleaseStatusPicker,
+} from '@liatrio/backstage-plugin-autogov-status-catalog-column';
+import { DefaultFilters } from '@backstage/plugin-catalog-react';
+```
+
 1. Further down, find the following code block and update:
 
+`diff`
```

Comment:
If this is indented with tabs to be sub-level to the above `1.`, the numbering sequence will be automatically continued.